### PR TITLE
Don't show paginator when search is being used

### PR DESCRIPTION
**Pull request by openjck**
The paginator doesn't do anything in this circumstance, so there's no
point in showing it.

### CODE REVIEW

File: src/components/containers/ExperimentContainer.js [hunk 1]
```diff
@@ -79,12 +79,14 @@ class ExperimentContainer extends React.Component {
         } else if (experimentFetch.rejected) {
             return <Error message={experimentFetch.reason.message} />;
         } else if (experimentFetch.fulfilled) {
-            let visibleMetricIds;
+            let visibleMetricIds, searchActive;
```
Comment (spasovski) [inline src/components/containers/ExperimentContainer.js]: JIT prefers initialization.

Comment (spasovski) [reply src/components/containers/ExperimentContainer.js]: Ah sorry should've been clearer. Providing an initial value so `let visibleMetricIds = [], searchActive = false` for example.

Comment (openjck) [reply src/components/containers/ExperimentContainer.js]: Gotcha. Updated. Do you also have a link to more info about that?

Comment (spasovski) [reply src/components/containers/ExperimentContainer.js]: I had a better one that I can no longer find but this explains it wrt to V8: https://www.html5rocks.com/en/tutorials/speed/v8/#toc-topic-numbers

**Comment by openjck** [comment]
Any other changes needed on this?

**Comment by spasovski** [comment]
ARRRR plus

**Comment by openjck** [comment]
☠️

**Pull request merged**